app/src/components/common/FormDate.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { ReactNode } from 'react';
+import styled from '@emotion/styled';
+
+const Styled = {
+  Wrapper: styled.div`
+    position: relative;
+    font-family: ${props => props.theme.fonts.work.light};
+    font-weight: 300;
+    font-size: ${props => props.theme.sizes.s};
+    color: ${props => props.theme.colors.offWhite};
+  `,
+  Input: styled.input`
+    color: ${props => props.theme.colors.offWhite};
+    background-color: ${props => props.theme.colors.overlay};
+    border-width: 0;
+    border-bottom: 1px solid ${props => props.theme.colors.gray};
+    padding: 5px 40px 5px 5px;
+    width: 100%;
+
+    &:active,
+    &:focus {
+      outline: none;
+      border-bottom-color: ${props => props.theme.colors.white};
+    }
+
+    &::placeholder {
+      color: ${props => props.theme.colors.gray};
+    }
+  `,
+  Extra: styled.div`
+    position: absolute;
+    top: 0;
+    right: 0;
+    background-color: transparent;
+    padding: 5px;
+  `,
+};
+
+interface Props {
+  label?: string;
+  value?: string;
+  extra?: ReactNode;
+  placeholder?: string;
+  className?: string;
+  onChange?: (value: string) => void;
+}
+
+const FormDate: React.FC<Props> = ({
+  label,
+  value,
+  placeholder,
+  extra,
+  className,
+  onChange,
+}) => {
+  const { Wrapper, Input, Extra } = Styled;
+  return (
+    <Wrapper className={className}>
+      <Input
+        type="date"
+        value={value}
+        onChange={e => onChange && onChange(e.target.value)}
```

Comment:
Do we want to use the `useCallback` format similar to what's used in Terminal?

Reply:
This was copied from the FormInput component. If we change it here we should probably update all of the input components to be consistent?